tools/package-release/README.md

```diff
@@ -0,0 +1,23 @@
+# Releasing Javascript packages
+
+When a package contains sufficient changes to justify a release to [NPM](https://www.npmjs.com/), follow these instructions to create a new release from the monorepo.
+
+## Prepare packages for release
+
+In order to prepare a package for release, a changelog will need to be compiled along with an appropriate version bump.
+
+1. Visit the monorepo Action [Prepare Package Release workflow](https://github.com/woocommerce/woocommerce/actions/workflows/prepare-package-release.yml). Click the "Run workflow" button.
+
+![image](https://user-images.githubusercontent.com/1922453/179434424-f08af974-5597-4c6f-955b-43faf062c7a7.png)
+
+2. Select the branch you'd like to release from. Typically, this would be `trunk`. You can also choose wich packages to prepare for release by supplying them as comma separated values in the input box. To loop through all packages, simply leave the `-a` flag. Packages that don't have any changes will be skipped and left as is.
```

Comment:
```suggestion
2. Select the branch you'd like to release from. Typically, this would be `trunk`. You can also choose which packages to prepare for release by supplying them as comma separated values in the input box. To loop through all packages, simply leave the `-a` flag. Packages that don't have any changes will be skipped and left as is.
```